mbed_build/_internal/templates/mbed_config.tmpl

```diff
@@ -14,6 +14,12 @@ set(MBED_PROFILE "develop" CACHE STRING "")
 set_property(GLOBAL APPEND PROPERTY MBED_TARGET_LABELS{% for label in labels %}
```

Comment:
Is it possible to move `{% for label in labels %}` to next line. It'll improve readability.

Reply:
It would. It leaves a blank line in the generated CMake if I do it, though. Let me check the jinja docs.

Reply:
I'm just going to leave it for now. The lstrip_block option messes up some of the macro definitions and this isn't that bad anyway :trollface: